tests/vm_state_test.go

```diff
@@ -161,10 +185,16 @@ var _ = Describe("[sig-storage]VM state", decorators.SigStorage, decorators.Requ
 			err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Delete(context.Background(), vm.Name, metav1.DeleteOptions{})
 			Expect(err).ToNot(HaveOccurred())
 		},
-			Entry("[test_id:10818]TPM across migration and restart", true, false, "migrate", "restart"),
-			Entry("[test_id:10819]TPM across restart and migration", true, false, "restart", "migrate"),
-			Entry("[test_id:10820]EFI across migration and restart", false, true, "migrate", "restart"),
-			Entry("[test_id:10821]TPM+EFI across migration and restart", true, true, "migrate", "restart"),
+			Entry("[test_id:10818]TPM across migration and restart", decorators.SigComputeMigrations, tpm, !efi, rwx, "migrate", "restart"),
+			Entry("[test_id:10819]TPM across restart and migration", decorators.SigComputeMigrations, tpm, !efi, rwx, "restart", "migrate"),
+			Entry("[test_id:10820]EFI across migration and restart", decorators.SigComputeMigrations, !tpm, efi, rwx, "migrate", "restart"),
+			Entry("[test_id:10821]TPM+EFI across migration and restart", decorators.SigComputeMigrations, tpm, efi, rwx, "migrate", "restart"),
+			// The entries below are clones of the entries above, but made for cluster that *do not* support RWX FS.
+			// They can't be flake-checked since the flake-checker cluster does support RWX FS.
+			Entry("TPM across migration and restart", decorators.SigCompute, decorators.NoFlakeCheck, tpm, !efi, rwo, "migrate", "restart"),
+			Entry("TPM across restart and migration", decorators.SigCompute, decorators.NoFlakeCheck, tpm, !efi, rwo, "restart", "migrate"),
+			Entry("EFI across migration and restart", decorators.SigCompute, decorators.NoFlakeCheck, !tpm, efi, rwo, "migrate", "restart"),
+			Entry("TPM+EFI across migration and restart", decorators.SigCompute, decorators.NoFlakeCheck, tpm, efi, rwo, "migrate", "restart"),
 		)
 		It("should remove persistent storage PVC if VMI is not owned by a VM", func() {
```

Comment:
Since you dropped `Sig*` label decorator at `Describe` level, this test is currently not being tested.

Reply:
We should adjust `check-unassigned-test.sh` script

Reply:
Really? Why is that? The test has a sig... What's different with `Describe()`?

Reply:
The test script uses label filtering instead of file name based filtering.
So, from the test script point of view the `[sig-compute]` on the root `Describe` means nothing. Each test should have at least one `Sig*` label decorator.
The mis-alignment is in the `check-unassigned-test.sh` which checks the `[sig-*]` presence in the test name descriptor https://github.com/kubevirt/kubevirt/blob/main/hack/check-unassigned-tests.sh while the relevant part is the presence of the label. I'm working on a patch for that.

Reply:
https://github.com/kubevirt/kubevirt/pull/13106

Reply:
Oh, that test, nice catch! I thought you were referring to the entries above. Thanks!